web/src/main/java/com/microsoft/applicationinsights/web/extensibility/modules/WebRequestTrackingTelemetryModule.java

```diff
@@ -117,7 +117,7 @@ public void onEndRequest(ServletRequest req, ServletResponse res) {
 
             ApplicationInsightsHttpResponseWrapper response = ((ApplicationInsightsHttpResponseWrapper)res);
             if (response != null) {
-                telemetry.setSuccess(response.getStatus() >= 200 && response.getStatus() < 300);
+                telemetry.setSuccess(response.getStatus() < 400);
```

Comment:
Thanks for the contribution.
I see the  .NET SDK [also treat 401 as success](https://github.com/Microsoft/ApplicationInsights-dotnet/blob/master/src/Core/Managed/Shared/DataContracts/RequestTelemetry.cs#L233)

Please keep the same behavior so the SDKs are aligned. Thanks.